CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Changelog
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+### [0.5.141](https://github.com/myhelix/cdk-watchful/compare/v0.5.140...v0.5.141) (2024-02-13)
```

Comment:
The latest tag is v0.5.203, may want to change this to be 204... I am genuinely confused how this version correlates to the npm package version as most repos use 2.29.0

https://www.npmjs.com/package/@myhelix/cdk-watchful?activeTab=versions